physionet-django/user/managers.py

```diff
@@ -1,30 +1,57 @@
-from django.db.models import DateTimeField, ExpressionWrapper, QuerySet, F, Q
+from django.db.models import (DateTimeField, ExpressionWrapper, QuerySet, F, Q,
+                              OuterRef, Subquery, Case, When)
 from django.utils import timezone
 
-from user.enums import TrainingStatus
-
+from user.enums import TrainingStatus, RequiredField
+from training.models import Course
 
 class TrainingQuerySet(QuerySet):
     def get_review(self):
-        return self.filter(status=TrainingStatus.REVIEW)
```

Comment:
please could you add docstrings for these methods to explain how they should be used?

Reply:
Sure, I've added the docstrings to the new one I added, and all the others - detailing what objects they will return.